lua/grapple-line.lua

```diff
@@ -156,8 +160,17 @@ local function get_name(path, depth)
 		table.insert(parts, part)
 	end
 
```

Comment:
I don't think `get_name` is the right function to change. This is just a utility function to build a name using a depth, so should always respect that depth.